src/brogue/RogueMain.c

```diff
@@ -1323,11 +1323,12 @@ void victory(boolean superVictory) {
     rogue.playbackMode = false;
     rogue.playbackMode = isPlayback;
 
+    displayMoreSign();
+
     if (serverMode) {
-        // There's no save recording prompt, so let the player see achievements.
-        displayMoreSign();
         saveRecordingNoPrompt(recordingFilename);
     } else {
+        blackOutScreen();
```

Comment:
so when we get a "supervictory" (mastery) does it snap to black and show dialog? oh wait it fades back to black beforehand doesn't it?

Reply:
i wasn't sure so i just checked in wizard mode. it's a "snap", not a fade. so the achievements screen gets blacked out, the the save dialog is presented, then the high scores, and then back to the title menu.

Reply:
Ok I just tested this. I misunderstood what was going on - this new version is a-ok